frontend_lib/test/component/NewShareDownload.spec.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import sinon from 'sinon'
+import { NewShareDownload } from '../../src/component/ShareDownload/NewShareDownload.jsx'
+require('../../src/component/ShareDownload/ShareDownload.styl')
+
+describe('<NewShareDownload />', () => {
+  const onChangeEmailsCallBack = sinon.stub()
+  const onKeyDownEnterCallBack = sinon.stub()
+  const onChangePasswordCallBack = sinon.stub()
+  const onClickCancelButtonCallBack = sinon.stub()
+  const onClickNewShareCallBack = sinon.stub()
+
+  const props = {
+    shareEmails: 'randomEmail@randomEmail.randomEmail',
+    onChangeEmails: onChangeEmailsCallBack,
+    onKeyDownEnter: onKeyDownEnterCallBack,
+    hexcolor: 'yellow',
+    sharePassword: 'randomPassword',
+    onChangePassword: onChangePasswordCallBack,
+    onClickCancelButton: onClickCancelButtonCallBack,
+    onClickNewShare: onClickNewShareCallBack,
+    emailNotifActivated: false
+  }
+
+  const wrapper = shallow(
+    <NewShareDownload
+      {...props}
+      t={key => key}
+    />
+  )
+
+  describe('Static design', () => {
+    it(`emails input should have the value: ${props.shareEmails}`, () => {
+      expect(wrapper.find('.shareDownload__email__input').prop('value')).to.equal(props.shareEmails)
+    })
+
+    it(`password input should have the value: ${props.sharePassword}`, () => {
+      wrapper.find('.shareDownload__password__link').simulate('click', { preventDefault: () => {}, stopPropagation: () => {} })
+      expect(wrapper.find('.shareDownload__password__input').prop('value')).to.equal(props.sharePassword)
+    })
+  })
+
+  describe('Handlers', () => {
+    it('onChangeEmails handler should call the proper handler', () => {
```

Comment:
replace "the proper handler" by it's name for each test.
The name is the props's name